src/apartments/repositories/apartment.repository.ts

```diff
@@ -0,0 +1,60 @@
+import { InjectRepository } from '@nestjs/typeorm';
+import { Injectable } from '@nestjs/common';
+import {
+  DeleteResult,
+  Repository,
+  SelectQueryBuilder,
+  UpdateResult,
+} from 'typeorm';
+
+import { CreateApartmentDto } from '../dto/create-apartment.dto';
+import { UpdateApartmentDto } from '../dto/update-apartment.dto';
+import { ApartmentEntity } from '../entities/apartment.entity';
+import { FindEntityDto } from '../../common/dto/find-entity.dto';
+
+@Injectable()
+export class ApartmentRepository {
+  public constructor(
+    @InjectRepository(ApartmentEntity)
+    private readonly repository: Repository<ApartmentEntity>,
+  ) {}
+
+  private createQueryBuilder({
+    limit,
+    skip,
+  }: FindEntityDto): SelectQueryBuilder<ApartmentEntity> {
+    return this.repository
+      .createQueryBuilder('apartment')
+      .take(limit)
+      .skip(skip);
+  }
```

Comment:
I believe that you need to include only those methods you are using in your service.